### PR TITLE
WIP Remove jenkins warning

### PR DESCRIPTION
**Pull request by pgdr**
@joakim-hove Can we remove the `/dev/random` and `/dev/urandom` from `util.c` and use `rand()`?

### CODE REVIEW

**Comment by joakim-hove** [comment]
> Can we remove the /dev/random and /dev/urandom from util.c and use rand()?

Well - we need some "random" way to seed the generator; just calling `rand( )` will create the same sequence every time.

File: lib/util/rng.c [hunk 1]
```diff
@@ -236,8 +219,8 @@ double rng_get_double( rng_type * rng ) {
 }
 
 int rng_get_int( rng_type * rng , int max_value ) {
-  rng_safe_cast( rng );
-  return rng->forward( rng->state ) % max_value;
+  int x = (int) util_dev_urandom_seed();
```
Comment (joakim-hove) [inline lib/util/rng.c]: Ehhhh - no no; the `rng_get_int( )` function should get *the next* integer in a deterministic fashion, from the stateful `rng_type` instance. The proposed changes will just get a "truly random" value - that is *not* what we want.

**Comment by joakim-hove** [comment]
Hmmm - must say I am bit skeptical to this one; the changes when it comes to RNG behavior are not just warning polish.

**Comment by joakim-hove** [comment]
What is wrong with `/dev/random` and `/dev/urandom`?

**Comment by pgdr** [comment]
removing this playground mumble, never intended to be seen

**Pull request closed without merging**